### PR TITLE
Fix lens reader arrows direction 

### LINKED ISSUE #104: Lens reader arrows not pointing in the right direction
- [x] Update the code to allow the configuration to update the arrow direction

## Steps

1. Determine whether the arrow direction problem is a code problem or a configuration problem by running the development environment and setting `COLLECT_POSITION=right` in your `config.env` file and see if the arrow directions change
1. If they don't change, have a look through the code to see where the bug might be: https://github.com/ACMILabs/playlist-label/search?q=COLLECT_POSITION
1. If we need a code change, create a pull-request with your code change
1. If we need a configuration change, let's pair program on making the change in XOS and push it to the Playlist Labels

### PR DESCRIPTION
**Pull request by samaSiriwardhanage**
Resolves #104 Lens reader arrows not pointing in the right direction.

### Acceptance Criteria
- [ ]  Update playlist label balena configuration to fix the arrow direction
- [ ] Update the code to allow the configuration to update the arrow direction

### Relevant design files
* None

### Testing instructions
1. None

### CODE REVIEW

File: app/static/playlist.style.css [hunk 1]
```diff
@@ -75,6 +75,10 @@ p:last-child {
   0% { -webkit-transform: translateX(0) rotate(90deg); }
   100% { -webkit-transform: translateX(-0.2em) rotate(90deg); }
 }
+@-webkit-keyframes animatedown {
+  0% { -webkit-transform: translateY(0) ; }
+  100% { -webkit-transform: translateY(0.2em) ; }
+}
```
Comment (sighmon) [inline app/static/playlist.style.css]:
```suggestion

@-webkit-keyframes animatedown {
  0% { -webkit-transform: translateY(0); }
  100% { -webkit-transform: translateY(0.2em); }
}
```

**Review by sighmon — CHANGES_REQUESTED** [review]
Looking good so far! Setting the `COLLECT_POSITION` to `right` and unsetting it works for me. But when I try and set it to `left` it still points down.

File: app/static/playlist.style.css (hunk 1 above)
Comment (samaSiriwardhanage) [reply app/static/playlist.style.css]: Changed code as suggested. Removed spaces.

**Review by sighmon — CHANGES_REQUESTED** [review]
Looks great! Thanks for fixing that `left` bug.

One small improvement might be to avoid having to duplicate a lot of the code in your new `.collect.left::before` style. So you could do something like this:

```css
.collect:before, .collect.left:before {
  width: 30px;
  height: 41px;
  display: inline-block;
  background: url('down.svg') center/contain no-repeat;
  content: " ";
  margin-right: 10px;
  transition: opacity 500ms;
  transform: rotate(90deg);
  vertical-align: -12px;
  -webkit-animation: animatedown 0.6s infinite alternate ease-in-out;
}

.collect.left:before {
  transform: rotate(-90deg);
  -webkit-animation: animateleft 0.6s infinite alternate ease-in-out;
}
```

**Review by sighmon — APPROVED** [review]
Looks great! I think this is ready to merge to `main`.

**Pull request merged**